app.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topicsController");
+
+
+const app = express();
+app.use(express.json());
```

Comment:
We're trying to write only the minimum amount of code needed to meet the requirements of the ticket, so I would take app.use(express.json()) out until you need it :)